Apps/WebClient/src/NewClientApp/src/components/TooManyRequestsComponent.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { useErrorStore } from "@/stores/error";
+import { computed } from "vue";
+
+interface Props {
+    location?: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+    location: "page",
+});
+
+const errorStore = useErrorStore();
+
+const warning = computed(() => errorStore.tooManyRequestsWarning);
```

Comment:
considering how the new Pinia stores are typed and using the values are simplified you could just consume this directly within the `showWarning` and `showError` computed variables.

```TypeScript
const showWarning = computed(() => errorStore.tooManyRequestsWarning == props.location);
const showError = computed(() => errorStore.tooManyRequestsError== props.location);
```

Reply:
Changed.